scio-extra/src/test/scala/com/spotify/scio/extra/hll/zetasketch/ZetasketchHllIntCounterTest.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.extra.hll.zetasketch
+
+import com.spotify.scio.testing.PipelineSpec
+import com.spotify.scio.testing.ApproximationAssertions._
+
+class ZetasketchHllIntCounterTest extends PipelineSpec {
+
+  "ZetasketchHLL++" should "estimate int distinct count" in {
+    val estimator = ZetasketchHllIntCounter()
+    val input = for (i <- 0 to 1000000) yield (i % 20)
+    val output = runWithData(input) { scl =>
+      scl
+        .countApproxDistinct(estimator)
+    }
+    output shouldApproximate withErrorRate(Seq(20L), 0.6d)
```

Comment:
Minor: we usually do `0.6f` for floats but leave out `d` for double. Is there a particular reason for this style?

Reply:
nope, just explicitly adding. will remove it.